### ISSUE TITLE
Open external URLs in a new tab

### ISSUE THREAD

**Issue by Dimitar5555** [deevroman/better-osm-org#25]
Based on https://github.com/openstreetmap/openstreetmap-website/issues/3567

I'm expanding it to notes, changesets, diaries and messages. It would be nice to also have [external URL icon](https://fontawesome.com/icons/arrow-up-right-from-square?f=classic&s=solid) after such URLs, so that users can see it's external with a glance and that it will open in a new tab.

**Comment by deevroman** [comment]
done in 0.4. I plan to publish a new version today / tomorrow, but you can test from the dev branch https://github.com/deevroman/better-osm-org/raw/dev/better-osm-org.user.js

p.s. I also don't like it so much that I open all the links in the browser by clicking on the mouse wheel or ctrl + click or use https://addons.mozilla.org/ru/firefox/addon/imagus-mod/ for images :)

**Comment by Dimitar5555** [comment]
Thank you! Indeed, it's a lot better. I was still hoping for the external URL icon after the link. Also it seems like it works only on notes and not in changesets (source tag on https://www.openstreetmap.org/changeset/153141789 or the link in the comment on https://www.openstreetmap.org/changeset/134606820) or links in messages.

**Comment by deevroman** [comment]
> Also it seems like it works only on notes and not in changesets

yes, I try to carefully change this, because somewhere opening a new tab is not necessary

> external URL icon

Something like 

<img width="123" alt="Снимок экрана 2024-06-25 в 16 17 22" src="https://github.com/deevroman/better-osm-org/assets/25708359/4e7d3a3d-4ec2-4174-914e-9a5485ef7673">

?

**Comment by Dimitar5555** [comment]
The icon is part of the text anchor and it can be added via CSS rules (https://stackoverflow.com/a/38529780/7849414). It doesn't matter if you click the icon or the text, the URL will open in a new tab regardless. See https://wiki.openstreetmap.org/wiki/Key:building#References for such example.

**Comment by deevroman** [comment]
I. e. do you mean to put an icon to show that the link will open in a new tab?

p. s. by the way, on the wiki* it looks like this icon only shows that the link leads to a third-party site and does not open it in a new tab

**Comment by Dimitar5555** [comment]
> I.e. do you mean to put an icon to show that the link will open in a new tab?

Yes.

> p.s. by the way, on the wiki* it looks like this icon only shows that the link leads to a third-party site and does not open it in a new tab

I always thought it meant both, but as it turns out, maybe I was wrong.

**Comment by deevroman** [comment]
fyi: since version 0.4.9, external links in object tags are also opened in a new tab.
